### PR TITLE
first pass at org github page

### PR DESCRIPTION
**Pull request by KwilLuke**
First draft for Kwil Github Organization page.

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+## Contributing
+
+Kwil welcomes pull requests on our public repositories. Please follow each repository's contributing guidelines.
+
+Open full time roles at Kwil are posted on our [careers page](https://www.kwil.com/careers).
+
+## Resources
+
+- 💻 [Website](https://www.kwil.com)
```
Comment (KwilLuke) [inline profile/README.md]: Yes or no on emojis?

**Review by jchappelow — COMMENTED** [review]
Super cool, but a 54MB gif is probably overkill.  :rofl:

**Comment by KwilLuke** [comment]
@jchappelow That's actually the most necessary part of the page.

Just kidding haha - I made a few edits and it got larger and larger. I can try to cut it back down.

File: profile/README.md (hunk 1 above)
Comment (KwilLuis) [reply profile/README.md]: I would move away from emojis - Vanesa made a good point that they don't add too much in the form of context with the downside of looking less refined

Comment (brennanjl) [reply profile/README.md]: I'd vote without.  Everything else looks good

**Pull request merged**